source/drivers/hal.cpp

```diff
@@ -87,6 +87,17 @@ err_t init_umbilical_uart() {
     MAP_GPIOPinConfigure(GPIO_PB1_U1TX);
     MAP_GPIOPinTypeUART(GPIO_PORTB_BASE, GPIO_PIN_0 | GPIO_PIN_1);
 
+    // Configure the UART for 115,200, 8-N-1 operation.
```

Comment:
Looks to be configured for 9600. Are we able to run the umbilical UART at 115200? Nothing on my end that I can think of.